internal/builders/generic/README.md

```diff
@@ -30,10 +30,20 @@ To get started, you will need to add some steps to your current workflow. We
 will assume you have an existing Github Actions workflow to build your project.
 
 Add a step to your workflow after you have built your project to generate a
-sha256 hash of your artifacts. The following assumes you have a binary called
-`binary-linux-amd64`.
+sha256 hash of your artifacts and base64 encode it.
 
-After that, add a new job to call the `slsa-github-generator` reusable workflow.
+Assuming you have a binary called `binary-linux-amd64` you can use the
+`sha256sum` and `base64` commands to create the digest. Here we use the `-w0` to
+output the encoded data on one line and make it easier to use as a Github Actions
+output:
+
+```shell
```

Comment:
maybe provide an example with multiple subjects, since most users won't be familiar with `sha256sum`?

Reply:
Done.